packages/app/src/cli/commands/app/config/use.ts

```diff
@@ -10,9 +12,20 @@ export default class ConfigUse extends Command {
   static flags = {
     ...globalFlags,
     ...appFlags,
+    reset: Flags.boolean({
+      hidden: false,
+      description: 'Reset current configuration.',
+      env: 'SHOPIFY_FLAG_RESET',
+      default: false,
+    }),
+  }
+
+  static args = {
+    config: Args.string(),
```

Comment:
we want to unify this arg with the one in `config push`, maybe extract to a separate module, and eventually move it to `appFlags`.

Reply:
+1 to Rachel's point. I don't think a `--config` flag makes sense for `use` because of cases like: `shopify app config use staging --config production`. It adds more confusion.